### PR TITLE
Code docs

### PR DESCRIPTION
**Pull request by kenthinson**
Some docs I wrote for today. I will continue as time allows.

### CODE REVIEW

**Comment by sh-dave** [comment]
Cool! Personally, i'd remove most of the `Optional (default is true/false)` and the type of return value though, as your IDE will already give you that info.

**Comment by kenthinson** [comment]
Thanks will fix it. Too bad spell check does not work in Kode Studio. I might have to write everything in a different software then copy and paste

**Comment by sh-dave** [comment]
Instead of KS, you can also use plain vscode with the `kha extension pack`. That way you can also install other extensions like a spellchecker. It's also possible in KS, but you have to  install the extensions manually.

**Comment by kenthinson** [comment]
@sh-dave Yeah the reason I have that optional info is for new programmers. Trying to make it noob friendly. I mean you can I can read the code and figure out what it's doing but I figured the docs should spell it out in plain English. What do you think of that idea?

**Comment by kenthinson** [comment]
@sh-dave  I'll try to get that working. Thanks :)

**Comment by sh-dave** [comment]
Hm, but you don't actually have to read the code in most cases (with the exception of `parent` parameter in `spawnObject` where i think mentioning it is actually good). And most newbies will probably use armory without even knowing iron or what it is to begin with. But i don't have that strong feelings about the comments being there, so do what you think is best 😃

**Comment by kenthinson** [comment]
@sh-dave thanks for the feedback :) I think I will air on the side of too verbose then on the side of not enough info. After all we can just ignore it and keep coding. Unless @luboslenco has a issue with that. I'm teaching my nephew how to program Using Armory. Seeing all his questions has really opened my eyes to how things I take for granted aren't to others
